examples/angular-todomvc/src/main.rs

```diff
@@ -65,11 +63,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .layer(layer),
         );
 
-    let server = Server::bind(&"0.0.0.0:8080".parse().unwrap()).serve(app.into_make_service());
-
-    if let Err(e) = server.await {
-        error!("server error: {}", e);
-    }
+    let listener = tokio::net::TcpListener::bind("0.0.0.0:3000").await.unwrap();
```

Comment:
port should be `8080`

Reply:
Can you detail a bit more ? If it is really the case, that is not the subject of this PR and it should be treated in another.

Reply:
The `socke.io` connects to the `8080` port. 
https://github.com/Totodore/socketioxide/blob/d563248f2b983af3ee3a25cff0850c4f3257d8d1/examples/angular-todomvc/dist/main.js#L297

Reply:
Or use CDN path